### ISSUE TITLE
Feature request: Allow the extension to run on Reddit Onion Service

### ISSUE THREAD

**Issue by antermin** [Geo1088/new-reddit-comment-faces#13]
[Reddit Onion Service was launched some months ago](https://old.reddit.com/r/redditsecurity/comments/yd6hqg/reddit_onion_service_launch/).

[Reddit Enhancement Suite also supported the Tor URL](https://old.reddit.com/r/Enhancement/comments/zqi9v8/the_new_release_of_res_is_safe_the_weird_looking/), but unfortunately some users were confused by the new URL permission requested when RES got updated, and as they did not grant the permission, the extension was disabled. They had to revert the change in the end.

However this extension seems to have only very few users (at the time of writing, AMO shows 8 users), so perhaps we don't have to worry too much.
Alternatively, a separate extension for Tor support can also be released.

**Comment by eritbh** [comment]
I agree that it's unlikely we'd have the same issue as RES had; I'm fine with adding support for the tor domain to the existing extension. I've only publicly talked about this extension a couple times so I'm relatively confident that anyone with concerns, or who denies the new permission and then realizes the extension isn't being activated anymore, will know where to contact me with issues.
